packages/labs/motion/demo/spring2d.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<!--
+  @license
+  Copyright 2021 Google LLC
+  SPDX-License-Identifier: BSD-3-Clause
+-->
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes"
+    />
+    <title>Spring 2D</title>
+    <script src="../development/demo/spring2d.js" type="module"></script>
+    <style>
+      html,
+      body,
+      goo-element {
+        width: 100%;
+        height: 100%;
+        margin: 0;
+        padding: 0;
+        background-color: white;
```

Comment:
background-color can optionally be removed, it is handled in the demo by the full screen element.